### PR TITLE
Add uncommited changeds to LocalGit

### PR DESCRIPTION
**Pull request by Soyn**
* details in https://github.com/danger/danger-js/issues/972

### CODE REVIEW

**Comment by orta** [comment]
Yeah,  there should already be a `--staging` flag in there this can use from when I first built this out

**Comment by Soyn** [comment]
I have updated the pull request.

File: source/dsl/cli-args.ts [hunk 1]
```diff
@@ -15,6 +15,8 @@ export interface CliArgs {
   dangerfile: string
   /** So you can have many danger runs in one code review */
   id: string
+  /** Use staged changes */
+  staged?: boolean
```
Comment (orta) [inline source/dsl/cli-args.ts]: Are you sure this works? The option above is `  .option("-s, --staging", "Just use staged changes.")` - so I'd expect this to be `staging`?

Comment (Soyn) [reply source/dsl/cli-args.ts]: In my local test. It works well. I will update the pull request later.

**Comment by Soyn** [comment]
Make `staged` to `staging`

File: source/commands/danger-local.ts [hunk 2]
```diff
@@ -16,16 +16,15 @@ interface App extends SharedCLI {
...
-  // .option("-s, --staging", "Just use staged changes.")
   .description("Runs danger without PR metadata, useful for git hooks.")
+  .option("-s, --staging", "Just use staged changes.")
   .option("-b, --base [branch_name]", "Use a different base branch")
 setSharedArgs(program).parse(process.argv)
 
 const app = (program as any) as App
 const base = app.base || "master"
 
-const localPlatform = new LocalGit({ base, staged: app.staging })
```
Comment (orta) [inline source/commands/danger-local.ts]: Oh, this is why it used to work - makes sense

**Comment by orta** [comment]
Cool beans, thanks!

**Pull request merged**